code/modules/surgery/organs/external/_external_organs.dm

```diff
@@ -39,12 +39,13 @@
 
 	///The color this organ draws with. Updated by bodypart/inherit_color()
 	var/draw_color
-
 	///Where does this organ inherit it's color from?
 	var/color_source = ORGAN_COLOR_INHERIT
 
 	///Does this organ have any bodytypes to pass to it's ownerlimb?
 	var/external_bodytypes = NONE
+	///Which flags does a 'modification tool' need to have to restyle us, if it all possible
```

Comment:
Comment where to find the flags for this

Reply:
Good point